redis/runtime/src/main/java/io/quarkiverse/langchain4j/redis/runtime/RedisEmbeddingStoreConfig.java

```diff
@@ -0,0 +1,83 @@
+package io.quarkiverse.langchain4j.redis.runtime;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.BUILD_AND_RUN_TIME_FIXED;
+
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkiverse.langchain4j.redis.MetricType;
+import io.quarkiverse.langchain4j.redis.VectorAlgorithm;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+/**
+ * Configuration of the Redis embedding store.
+ */
+@ConfigRoot(phase = BUILD_AND_RUN_TIME_FIXED)
+@ConfigMapping(prefix = "quarkus.langchain4j.redis")
+public interface RedisEmbeddingStoreConfig {
+
+    /**
+     * The name of the Redis client to use. These clients are configured by means of the `redis-client` extension.
+     * If unspecified, it will use the default Redis client.
+     */
+    @WithDefault("<default>")
+    String redisClientName();
```

Comment:
I think it's better to make this `Optional` and not have the scary `<default>` value visible in the Javadoc

Reply:
+1

Reply:
I'm also not sure about the name. The property will be "quarkus.langchain4j.redis.redis-client-name", we can remove the second "redis".